apps/src/redux/pageConstants.js

```diff
@@ -131,3 +132,19 @@ export function setPageConstants(props) {
     props: props
   };
 }
+
+/**
+ * Helpers
+ */
+
+function rootState() {
+  return getStore().getState()?.pageConstants;
+}
+
+export function hideRunButton() {
+  return rootState()?.hideRunButton;
```

Comment:
Why optional chaining? What would cause `rootState()` to be undefined? And is defaulting to `false` the expected behavior of `hideRunButton()` in that case?

Reply:
`rootState()` could be undefined if 1) our redux store has no state, or 2) you call this helper function but the `pageConstants` reducer was never registered -- both of which are possible but pretty unlikely.

this will default to `undefined` in that case, which is expected